packages/fxa-payments-server/src/components/PaymentErrorView/index.tsx

```diff
@@ -85,17 +85,20 @@ export const PaymentErrorView = ({
     <>
       {title}
       <section
-        className={`container card payment-error ${className}`}
+        className={`payment-error row-start-3 row-end-4 mt-6 mx-4 tablet:mt-0 ${className}`}
```

Comment:
Aligned with `payment-error row-start-3 row-end-4 tablet:mt-0`.

- Is `mx-4` from `.card`?

- Where is `mt-6` from? Should it be `mt-3` from:
```
[@]include max-width('tablet'){
  margin-top: 10px;
}
```

Reply:
Oh I forgot to delete this. Sorry!